cmd/upgrade.go

```diff
@@ -223,12 +215,12 @@ func (uc *upgradeCmd) loadCluster(cmd *cobra.Command) error {
 			parametersKey, nameSuffixKey, defaultValueKey, templatePath)
 	}
 
-	log.Infoln(fmt.Sprintf("Name suffix: %s", uc.nameSuffix))
+	log.Infoln(fmt.Sprintf("Upgrading cluster with name suffix: %s", uc.nameSuffix))
 
-	uc.agentPoolsToUpgrade = []string{}
-	log.Infoln(fmt.Sprintf("Gathering agent pool names..."))
+	uc.agentPoolsToUpgrade = make(map[string]bool)
+	uc.agentPoolsToUpgrade[kubernetesupgrade.MasterPoolName] = true
```

Comment:
Is this additive? I don't see us adding the master in the previous `agentPoolsToUpgrade` string slice pattern

Reply:
maybe I should rename this to `poolsToUpgrade` - the `agentPoolsToUpgrade` name doesn't really make sense